code/game/area/areas.dm

```diff
@@ -6,6 +6,9 @@
 #define VOLUME_AMBIENT_HUM 18
 #define VOLUME_MUSIC 30
 
+/// This list of names is here to make sure we don't state our descriptive blurb to a person more than once.
+var/global/list/area_blurb_stated_to = list()
```

Comment:
I don't think this var have a reason to be global, it can just aswell be assigned to a client or mob, then the area category added to it on stating the blurb, and likewise the check done, without the need of a global list

Reply:
Uhh I'm not sure how to make it otherwise this was a port for a fix of the issue from elsewhere

Reply:
> Uhh I'm not sure how to make it otherwise this was a port for a fix of the issue from elsewhere

Then you should provide the appropriate credits.